client/apps/edit/components/admin/icons.jade

```diff
@@ -0,0 +1,2 @@
+.caret
+  include ../../public/icons/edit_caret_down.svg
```

Comment:
Might be nice to consolidate icons into one folder at some point. I think stuff like a caret down arrow might be useful in `components/layout/public/icons` eventually. Def not a blocker tho